failure on creating ubuntu image due to missing OVMF_VARS.fd

Hello,

When i try to generate an Ubuntu image by doing:

```
cd ubuntu
make
```

I run into this error:

```
rm -f -rf output-* custom-*.gz \
	seeds-flat.iso seeds-lvm.iso seeds-cloudimg.iso \
	OVMF_VARS.fd
packer init . && packer build \
	-only='cloudimg.*' \
	-var ubuntu_series=jammy \
	-var architecture=amd64 \
	-var ovmf_suffix=_4M \
	-var timeout=1h .
cloudimg.deps.null.dependencies: output will be in this color.
cloudimg.image.qemu.cloudimg: output will be in this color.

==> cloudimg.image.qemu.cloudimg: Retrieving ISO
==> cloudimg.image.qemu.cloudimg: Trying https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img
==> cloudimg.image.qemu.cloudimg: Trying https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img?checksum=sha256%3A55c687a9a242fab7b0ec89ac69f9def77696c4e160e6f640879a0b0031a08318
==> cloudimg.deps.null.dependencies: Running local shell script: /tmp/packer-shell831068070
==> cloudimg.deps.null.dependencies: cp: cannot stat '/usr/share/OVMF/OVMF_VARS.fd': No such file or directory
==> cloudimg.deps.null.dependencies: Provisioning step had errors: Running the cleanup provisioner, if present...
Build 'cloudimg.deps.null.dependencies' errored after 33 milliseconds 397 microseconds: Script exited with non-zero exit status: 1. Allowed exit codes are: [0]
==> cloudimg.image.qemu.cloudimg: https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img?checksum=sha256%3A55c687a9a242fab7b0ec89ac69f9def77696c4e160e6f640879a0b0031a08318 => /home/ubuntu/.cache/packer/a6b15540ad14c44efcb3b770426a6b3c090f7af9.iso
==> cloudimg.image.qemu.cloudimg: Creating required virtual machine disks
==> cloudimg.image.qemu.cloudimg: Resizing hard drive...
==> cloudimg.image.qemu.cloudimg: Starting HTTP server on port 8561
==> cloudimg.image.qemu.cloudimg: Found port for communicator (SSH, WinRM, etc): 3948.
==> cloudimg.image.qemu.cloudimg: Creating temporary RSA SSH key for instance...
==> cloudimg.image.qemu.cloudimg: Looking for available port between 5900 and 6000 on 127.0.0.1
==> cloudimg.image.qemu.cloudimg: Starting VM, booting disk image
    cloudimg.image.qemu.cloudimg: The VM will be run headless, without a GUI. If you want to
    cloudimg.image.qemu.cloudimg: view the screen of the VM, connect via VNC without a password to
    cloudimg.image.qemu.cloudimg: vnc://127.0.0.1:5971
==> cloudimg.image.qemu.cloudimg: Overriding default Qemu arguments with qemuargs template option...
==> cloudimg.image.qemu.cloudimg: Error launching VM: Qemu failed to start. Please run with PACKER_LOG=1 to get more info.
==> cloudimg.image.qemu.cloudimg: Deleting output directory...
Build 'cloudimg.image.qemu.cloudimg' errored after 1 second 244 milliseconds: Build was halted.

==> Wait completed after 1 second 244 milliseconds

==> Some builds didn't complete successfully and had errors:
--> cloudimg.deps.null.dependencies: Script exited with non-zero exit status: 1. Allowed exit codes are: [0]
--> cloudimg.image.qemu.cloudimg: Build was halted.

==> Builds finished but no artifacts were created.
make: *** [Makefile:46: custom-cloudimg.tar.gz] Error 1
```

My setup:

- Ubuntu OS : Noble 24.04
- I install packer following https://developer.hashicorp.com/packer/install#linux

--- a/ubuntu/ubuntu-cloudimg.pkr.hcl
+++ b/ubuntu/ubuntu-cloudimg.pkr.hcl
@@ -66,7 +66,7 @@ build {
 
   provisioner "shell-local" {
     inline = [
-      "cp /usr/share/${lookup(local.uefi_imp, var.architecture, "")}/${lookup(local.uefi_imp, var.architecture, "")}_VARS.fd ${lookup(local.uefi_imp, var.architecture, "")}_VARS.fd",
+      "cp /usr/share/${lookup(local.uefi_imp, var.architecture, "")}/${lookup(local.uefi_imp, var.architecture, "")}_VARS${var.ovmf_suffix}.fd ${lookup(local.uefi_imp, var.architecture, "")}_VARS.fd",
       "cloud-localds seeds-cloudimg.iso user-data-cloudimg meta-data"
     ]
     inline_shebang = "/bin/bash -e"

